open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/package-lock.json

```diff
@@ -0,0 +1,12822 @@
+{
```

Comment:
Is this a generated file?

Reply:
yes.
npm recommends to be committed
https://docs.npmjs.com/files/package-lock.json